chrome/edge-navigation.json

```diff
@@ -0,0 +1,43 @@
+{
+  "id": "edge",
+  "title": "Edge Management",
+  "navItems": [
+    {
+      "title": "Inventory",
+      "filterable": false,
```

Comment:
@resoluteCoder  I don't think this should be set to false. It will remove the items form the app filter. There was this PR to add it: https://github.com/RedHatInsights/cloud-services-config/pull/1117 and a ticket: https://issues.redhat.com/browse/RHCLOUD-18752

Reply:
@Hyperkid123 You are absolutely correct. Here is the PR to remove those filter properties https://github.com/RedHatInsights/cloud-services-config/pull/1153

Thanks for the lookout! :smile: